Source/Noesis.Javascript/JavascriptInterop.cpp

```diff
@@ -537,15 +541,16 @@ JavascriptInterop::ConvertFromSystemDictionary(System::Object^ iObject)
 v8::Local<v8::Value>
 JavascriptInterop::ConvertFromSystemList(System::Object^ iObject) 
 {
-	v8::Isolate *isolate = JavascriptContext::GetCurrentIsolate();
+	auto isolate = JavascriptContext::GetCurrentIsolate();
+	auto context = isolate->GetCurrentContext();

```

Comment:
Is there any doc about this context now being required everywhere?

Reply:
There is never a doc about these breaking changes unfortunately. You basically can only look at the errors and the new signatures... If you find one, let me know 😃